plugins/nf-ga4gh/src/main/nextflow/ga4gh/tes/executor/TesFileCopyStrategy.groovy

```diff
@@ -113,6 +113,7 @@ class TesFileCopyStrategy implements ScriptFileCopyStrategy {
             copy.remove('PATH')
         // when a remote bin directory is provide managed it properly
         if( remoteBinDir ) {
+            result << "mkdir \$PWD/nextflow-bin/\n"
```

Comment:
This line resolves an error we were encountering when attempting to run the `.command.run` file within the container:

```sh
root@<container id>:/work $ ./.command.run
cp: cannot create regular file '/work/nextflow-bin/': Not a directory
```